cgosqlite/cgosqlite.go

```diff
@@ -31,6 +31,14 @@ package cgosqlite
 // #cgo linux LDFLAGS: -ldl -lm -lrt
 // #cgo linux CFLAGS: -std=c99
 //
+// // TODO(crawshaw): for some reason, I cannot get CLOCK_MONOTONIC
+// // defined properly here by cgo. My C compiler seems otherwise
+// // fine, I can compile a small C program on linux referring to
+// // CLOCK_MONOTONIC.
+// //
+// // For now, use the value directly. It's fixed on linux.
```

Comment:
what does "It's fixed on linux" mean?

oh! as in "it's a fixed value on linux".

I thought there was a bug being referred to.